app/scripts/directives/bindService.js

```diff
@@ -71,6 +71,7 @@ function BindService($scope,
                                         .concat(replicaSets)
                                         .concat(statefulSets);
       ctrl.applications = _.sortByAll(apiObjects, ['metadata.name', 'kind']);
```

Comment:
There are three options: 
- 'true' = bind to a selected application
- 'false' = bind, but not to an application
- 'none' = do not bind
-